addons/mail/static/src/messaging/component/follower_subtype_list/follower_subtype_list.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.FollowerSubtypeList" owl="1">
+        <div class="o_FollowerSubtypeList modal-dialog">
+            <t t-if="followerSubtypeList">
+                <div class="modal-content">
+                    <header class="modal-header">
+                        <h4 class="modal-title">
+                            Edit Subscription of <t t-esc="followerSubtypeList.follower.name"/>
+                        </h4>
+                        <i class="close fa fa-times" data-dismiss="modal" aria-label="Close" t-on-click="_onClickCancel"/>
+                    </header>
+                    <main class="modal-body">
+                        <div class="o_FollowerSubtypeList_subtypes">
+                            <t t-foreach="followerSubtypeList.follower.subtypes" t-as="subtype" t-key="subtype.id">
+                                <FollowerSubtype
+                                    class="o_FollowerSubtypeList_subtype"
+                                    followerLocalId="followerSubtypeList.follower.localId"
+                                    followerSubtypeLocalId="subtype.localId"
+                                />
+                            </t>
+                        </div>
+                    </main>
+                    <div class="modal-footer">
+                        <button class="o-apply btn btn-primary" t-on-click="_onClickApply">
+                            Apply
+                        </button>
+                        <button class="o-cancel btn btn-secondary" t-on-click="_onClickCancel">
+                            Cancel
+                        </button>
+                    </div>
+                </div>
+            </t>
+        </div>
+    </t>
```

Comment:
We really have to use bootstrap modal manually in this file?

Reply:
Maybe not, it's just to ease the styling I think. You would do the CSS without using it ?

Reply:
It is just for the style? `data-dismiss="modal"` etc seems necessary for JS stuff, if we handle the click in OWL I think they should be removed if possible.

Otherwise I'm OK if we use bootstrap classes for styling when it is for more complex elements like this.